core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -116,11 +116,11 @@ public final String getSearchUrl() {
      * Mark this monitor as disabled, to prevent this from showing up in the UI.
      */
     public void disable(boolean value) throws IOException {
-        AbstractCIBase hudson = Jenkins.getInstance();
-        Set<String> set = hudson.disabledAdministrativeMonitors;
+        AbstractCIBase jenkins = Jenkins.get();
```

Comment:
just `Jenkins`

Reply:
Nope. "disabledAdministrativeMonitors" below is a package-scope method, so you need a class from a proper scope to access it. Didn't want to refactor too much

Reply:
OK, though you could make it `@Restricted(NoExternalUse.class) public`.

Maybe worth a comment to the effect that this is bypassing package access.